src/briefcase/platforms/android/gradle.py

```diff
@@ -282,6 +282,24 @@ def permissions_context(self, app: AppConfig, x_permissions: dict[str, str]):
             "features": features,
         }
 
+    def _write_requirements_file(
+        self, app: AppConfig, requires: list[str], requirements_path: Path
+    ):
+        super()._write_requirements_file(app, requires, requirements_path)
+
+        # Flatpak runs ``pip install`` using an ``install_requirements.sh`` which Briefcase uses
```

Comment:
See comment above about line length.

Reply:
Fixed in https://github.com/beeware/briefcase/pull/2059/commits/2807664b3406436f505a181f8367761b9d10543e